### PR TITLE
covert those two paragraphs into a bulleted list is done

### LINKED ISSUE #18187: First sentence about only deleting issues in a personal repository is misleading
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/issues/tracking-your-work-with-issues/deleting-an-issue

### What part(s) of the article would you like to see updated?

> You can only delete issues in a repository owned by your personal account. You cannot delete issues in a repository owned by another personal account, even if you are a collaborator there.

I read that first sentence as meaning "it is not at all possible to delete issues that are not in personal accounts"

I'd suggest changing the paragraph to:

> If an issue is in a personal account, you can only delete it if it is your personal account.

or

> If an issue is in a personal account, it can only be deleted by that account's owner. 

I understand that collaborators are things. But, I'm not sure that mentioning them is relevant if the wording is sufficiently clear. -- the second form would probably give space to mention collaborators if someone felt a real need.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by bishal7679**
### Why:

Closes https://github.com/github/docs/issues/18187

### What's being changed:
covert those two paragraphs into a bulleted list is done

Screenshot (130)

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by japui** [comment]
issues/tracking-your-work-with-issues/deleting-an-issue.md

**Comment by bishal7679** [comment]
@japui what do you mean by that?

File: content/issues/tracking-your-work-with-issues/deleting-an-issue.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ versions:
 topics:
   - Pull requests
 ---
-You can only delete issues in a repository owned by your personal account. You cannot delete issues in a repository owned by another personal account, even if you are a collaborator there.
+The permissions required depend on whether the repository is owned by your personal account or an organization.
+- You can only delete issues in a repository owned by your personal account.
+- You cannot delete issues in a repository owned by another personal account, even if you are a collaborator there.
```
Comment (skedwards88) [inline content/issues/tracking-your-work-with-issues/deleting-an-issue.md:19]:
```suggestion
- You cannot delete issues in a repository owned by an organization or another personal account, even if you are a collaborator there.
```

**Review by skedwards88 — COMMENTED** [review]
Thanks for this improvement! I made a small wording suggestion that I'll incorporate, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

File: content/issues/tracking-your-work-with-issues/deleting-an-issue.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ versions:
 topics:
   - Pull requests
 ---
-You can only delete issues in a repository owned by your personal account. You cannot delete issues in a repository owned by another personal account, even if you are a collaborator there.
+The permissions required depend on whether the repository is owned by your personal account or an organization.
```
[the remark this answers is not in the document]
Comment (bishal7679) [reply content/issues/tracking-your-work-with-issues/deleting-an-issue.md]: Looks awesome! Thank you for this changes⚡🎉

**Pull request merged**